.github/workflows/ci.yml

```diff
@@ -49,7 +49,7 @@ jobs:
         name: Set up Python
         uses: actions/setup-python@v4
         with:
-          python-version: 3.9
+          python-version: 3.8
```

Comment:
Any reason this downgraded?

Reply:
This was my workaround for the fact that when you use 3.9 it doesnt install zipp (and fails), because I noticed https://github.com/paperless-ngx/paperless-ngx/blob/6deb2925ce9e787d183c22db229177cbfcb65ca3/Pipfile.lock#L2800

If theres a better way to fix e.g. the lock file of course lmk!

Reply:
Do you have a failing run I can look at?  The command `pipenv run mkdocs build` works fine on 3.10, which doesn't install zipp either.

Reply:
Yea, just changed back to 3.9 and now see https://github.com/paperless-ngx/paperless-ngx/actions/runs/3605664039/jobs/6076281867

Reply:
Ps. changing to 3.10 failed with something like that python version not being supported

Reply:
There's something really weird about those 2 packages.  Guess leave it at 3.8

Reply:
Haha, ok, hope it doesnt drive you crazy to see it there 😅